flytekit/control_plane/nodes.py

```diff
@@ -0,0 +1,310 @@
+
+    @property
+    def upstream_nodes(self) -> List["FlyteNode"]:
+        return self._upstream
+
+    @property
+    def upstream_node_ids(self) -> List[str]:
+        return list(sorted(n.id for n in self.upstream_nodes))
+
+    @property
+    def outputs(self) -> Dict[str, _promise.NodeOutput]:
+        return self._outputs
+
+    def assign_id_and_return(self, id: str):
+        if self.id:
+            raise _user_exceptions.FlyteAssertion(
+                f"Error assigning ID: {id} because {self} is already assigned. Has this node been ssigned to another "
+                "workflow already?"
+            )
+        self._id = _dnsify(id) if id else None
+        self._metadata.name = id
+        return self
+
+    def with_overrides(self, *args, **kwargs):
+        # TODO: Implement overrides
+        raise NotImplementedError("Overrides are not supported in Flyte yet.")
+
+    @_exception_scopes.system_entry_point
+    def __lshift__(self, other: "FlyteNode") -> "FlyteNode":
+        """
+        Add a node upstream of this node without necessarily mapping outputs -> inputs.
+        :param other: node to place upstream
+        """
+        if hash(other) not in set(hash(n) for n in self.upstream_nodes):
+            self._upstream.append(other)
+        return other
+
+    @_exception_scopes.system_entry_point
+    def __rshift__(self, other: "FlyteNode") -> "FlyteNode":
+        """
+        Add a node downstream of this node without necessarily mapping outputs -> inputs.
+
+        :param other: node to place downstream
+        """
+        if hash(self) not in set(hash(n) for n in other.upstream_nodes):
+            other.upstream_nodes.append(self)
+        return other
+
+    def __repr__(self) -> str:
+        return f"Node(ID: {self.id} Executable: {self._executable_flyte_object})"
+
+
+class FlyteNodeExecution(_node_execution_models.NodeExecution, _artifact_mixin.ExecutionArtifact):
+    def __init__(self, *args, **kwargs):
+        super(FlyteNodeExecution, self).__init__(*args, **kwargs)
+        self._task_executions = None
+        self._workflow_executions = None
+        self._inputs = None
+        self._outputs = None
+
+    @property
+    def task_executions(self) -> List["flytekit.control_plane.tasks.executions.FlyteTaskExecution"]:
+        return self._task_executions or []
+
+    @property
+    def workflow_executions(self) -> List["flytekit.control_plane.workflow_executions.FlyteWorkflowExecution"]:
+        return self._workflow_executions or []
+
+    @property
+    def executions(self) -> _artifact_mixin.ExecutionArtifact:
+        return self.task_executions or self.workflow_executions or []
+
+    @property
+    def inputs(self) -> Dict[str, Any]:
+        """
+        Returns the inputs to the execution in the standard python format as dicatated by the type engine.
+        """
+        if self._inputs is None:
```

Comment:
let's just ignore everything and just return flyte literals for now.  kill all the old code.